README.md

```diff
@@ -28,7 +28,7 @@ ysoserial.net generates deserialization payloads for a variety of .NET formatter
 
 Available gadgets:
 
-	ActivitySurrogateDisableTypeCheck (Disables 4.8+ type protections for ActivitySurrogateSelector, command is ignored.)
+	ActivitySurrogateDisableTypeCheck (Disables 4.8+ type protections for ActivitySurrogateSelector, command is ignored.) [derived method]
```

Comment:
What do you mean by [derived method]?

Reply:
These are the methods that use full or part of a gadget that has been found before. Very close meaning to the bridged method.

Reply:
ok, thanks for clarifying